templates/CHP.iesopt.template.yaml

```
@@ -0,0 +1,221 @@
+parameters:
+  mode: ~                           # operation mode: extraction, condensing, backpressure
```

```suggestion
  mode: ~                           # operation mode: extraction (full characteristic field), condensing (power only), backpressure (acc. to power ratio line)
```
I would like that but maybe too crowded then.